src/ntfsck.c

```diff
@@ -382,10 +383,16 @@ static int ntfsck_check_runlist(ntfs_attr *na, u8 set_bit, struct rl_size *rls,
 	}
 
 	if (dup_rl) {
-		check_failed("Found cluster duplication of inode(%"PRId64":%d), Fix it\n",
```

Comment:
you replace check_failed() with ntfs_log_error(). If so, fsck_err_found() is not called. no problem ?

Reply:
there is some consideration about this point.

check_failed() is called outside of this function(ntfsck_check_runlist())
because ntfsck_check_attr_runlist() check two kinds of corruption which runlist corruption and cluster duplication with ntfsck_decompose_runlist() and ntfsck_check_runlist() respectively, but applying repaired cluster run to disk is one place outside of ntfsck_check_attr_runlist().

If we count error inside function, error may be counted up to 2. but just error fixing occurred one place, it due to decrease just one error count. So, even though fsck fixed all error, fsck misundertood error is remained

I don't like this style of implementation. if you have any idea, let me know.